Formula/flif.rb

```diff
@@ -2,10 +2,10 @@ class Flif < Formula
   desc "Free Loseless Image Format"
   homepage "https://flif.info/"
   # When updating, please check if FLIF switched to CMake yet
```

Comment:
I think they use CMake now. You can try
```ruby
system "cmake", "-S", "src", "-B", "build"
system "cmake", "--build", "build"
system "cmake", "--install", "build"
```

Reply:
Tried in my local, does not build successfully.

>   file INSTALL cannot copy file
  "/tmp/flif-20211206-95109-192d1ig/FLIF-0.4/build/libflif.0.dylib" to
  "/usr/local/lib/libflif.0.dylib": Operation not permitted.

Reply:
oops, that was supposed to be
```ruby
system "cmake", "-S", "src", "-B", "build", *std_cmake_args
system "cmake", "--build", "build"
system "cmake", "--install", "build"
```